server/assets-v1/templates/src/pages/client_portal/register.html

```diff
@@ -160,18 +163,19 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
       const isBUFocused = ref(false);
       const isUsernameFocused = ref(false);
       const isPasswordFocused = ref(false);
-      const client = ref({
-        id: "",
-        secret: "",
-        name: "",
-        redirect_uri: "",
-        backend_uri: "",
-      });
+      const showToast = ref(false);
+      const toastMessage = ref("");
+      const backend_uri_text = ref("");
```

Comment:
incosistency, other variable is using camelCase but this variable is using snake_case